pages/common/st-util.md

```diff
@@ -0,0 +1,15 @@
+# st-util
+
+> Run GDB server to interact with STM32 ARM Cortex microcontoller.
```

Comment:
Sorry, I had added a comment clarifying what @agnivade meant, but it was pending (still not used to this new review system 😅)